src/website/views/components/brand.pug

```diff
@@ -0,0 +1,61 @@
+---
+title: Brand
+---
+
+- var sizes = ['xs', 'sm', 'sm2', 'sm3', 'md', 'lg', 'xl', 'xxl'];
+
+p.-text Brand is the graphic symbol that represents the company or organization.
+
+h3 Examples
+p.-text
+  | To render the brand, use the class <code>a-brand</code>.
+
+h4 Base
+.example.-mb3
+  .-p3
+    a.a-brand(href="#", aria-label="CenturyLink")
+      img(src='../../assets/images/centurylink-logo.svg', alt="CenturyLink")
+  :code(lang="html")
+    <a class="a-brand" href="#" aria-label="CenturyLink">
+      <img src="../../assets/images/centurylink-logo.svg" alt="CenturyLink">
```

Comment:
Relative path will not work. We need to show the absolute path in all examples so developers can copy an actual image.

Reply:
You are right, but we have a problem each time we compile a new version the url changes.  https://assets.ctl.io/chi/0.8.2/... So currently i can not put absolute path